cloud/src/meta-service/meta_service.cpp

```diff
@@ -1849,6 +1849,88 @@ static bool remove_pending_delete_bitmap(MetaServiceCode& code, std::string& msg
     return true;
 }
 
+static bool check_partition_version_when_update_delete_bitmap(
```

Comment:
add detailed comment for this function
what is the purpose and what it checks  1 2 3 4 list all